attributes/default.rb

```diff
@@ -221,6 +221,11 @@
 # extra_packages to install
 default['datadog']['extra_packages'] = {}
 
+# extra config options
+# If an agent is released with a new config option which is not yet supported by this cookbook
+# you can use this attribute to set it. Will be ignored if falsy.
```

Comment:
I think we should ignore the value _only if_ it's `nil`. We have agent configuration options that are `true` by default so setting them to `false` is meaningful. Thoughts?